t04_refactored.py

```diff
@@ -245,9 +245,35 @@ def team_6_adv():
 
 
 def team_7_adv():
-    pass
-    # TODO Add your code here
+    """
+    https://docs.google.com/document/d/1ksWilu6oBRWPoa5kZo2jyzkAOiAKqwtHYfMWGK7d27M/edit
+    Hira Karim
+    Claudia Pulido
+    :return:
+    """
+    direction = input("Go with him further [Up, or Down]?")
+    if direction == "Up":
+        print("You've reached a large vault... a large console stands to its right...")
+        print("The console lights up when you reach it... it asks for a passcode")
+    else:
+        if direction == "Down":
+            print("you have reached the strongest creature in all of the lands..")
+            sleep(delay * 3)
+            print("It is the Teacher Assistant Silas! Do you fight or run?")
+    answer = input("Ask for help or guess the code")
+    if answer == "Ask for help":
+        print("You ask Dr. Heggen for help")
+        print("He pulls out a small electrical device and plugs it into the console")
+        print("The console glows green and the vault opens up")
+        print("you've found riches beyond your wildest dreams")
+    elif "Guess the code":
+        print("You type in '12345' and the console turns red")
+        print("The walls close in on you, crushing you like a Wookie in a trash compactor")
+    if dead == dead:
```

Comment:
This code always runs because dead is always equal to dead. But, you don't need it because the `kill_if_dead() `function call replaced it.